simplesource-data/src/main/java/io/simplesource/data/CommandError.java

```diff
@@ -0,0 +1,158 @@
+package io.simplesource.data;
```

Comment:
`CommandError` doesn't belong in the data module.
I think it should be in the command-api module.